BrainPortal/app/models/cluster_task.rb

```diff
@@ -2330,17 +2330,27 @@ def singularity_commands(command_script)
     # Numbers in (paren) correspond to the comment
     # block in the script, well below.
 
+    # (6) The path to the task's work directory
+    task_workdir  = self.full_cluster_workdir # a string
+
     # (1) additional singularity execution command options defined in ToolConfig
     container_exec_args = self.tool_config.container_exec_args.presence
 
-    # (2) The root of the shared area for all CBRAIN tasks
-    gridshare_dir = self.bourreau.cms_shared_dir
-
-    # (3) The root of the DataProvider cache
-    cache_dir     = self.bourreau.dp_cache_dir
+    # (2) The root of the DataProvider cache
+    cache_dir      = self.bourreau.dp_cache_dir
+    gridshare_dir  = self.bourreau.cms_shared_dir # not mounted explicitely
+    cache_dir_syml = "#{gridshare_dir}/DP_Cache"
 
-    # (6) The path to the task's work directory
-    task_workdir  = self.full_cluster_workdir
+    # (3) Ext3 capture mounts, if any.
+    # These will look like "-B .capt_abcd.ext2:/path/workdir/abcd:image-src=/"
```

Comment:
are you sure it is ext2 not ext3? (Not like I hold a grudge toward ext2 or ext4, who knowns, they might work as well, it is just everywhere else you say ext3

Reply:
It's an error in the comment. It's ext3. ext2 is not supported by Singularity anyway.